packages/api/lambdas/db-indexer.js

```diff
@@ -199,11 +199,7 @@ async function indexRecords(records) {
   return pMap(
     records,
     (record) => indexRecord(esClient, record).catch(log.error),
-    {
-      concurrency: process.env.ES_CONCURRENCY
-        ? Number.parseInt(process.env.ES_CONCURRENCY, 10)
-        : 3,
-    }
+    { concurrency: 1 }
```

Comment:
Sorry, I'm not trying to nitpick here, but can we just remove the use of `pMap` altogether? I worry that with using a library that supports concurrency, it will be easy for someone to accidentally re-enable it and re-introduce the bug.

Maybe use https://github.com/sindresorhus/p-each-series instead?

Reply:
yeah.  It's more clear.
[a26302c](https://github.com/nasa/cumulus/pull/1970/commits/a26302c059d2fbf46db3b6cf228576f9262b1032)